zingolib/src/wallet/transaction_records_by_id.rs

```diff
@@ -21,8 +31,94 @@ impl std::ops::DerefMut for TransactionRecordsById {
     }
 }
 impl TransactionRecordsById {
+    // initializers
+
     // Associated function to create a TransactionRecordMap from a HashMap
+    pub fn default() -> Self {
+        TransactionRecordsById(HashMap::new())
+    }
     pub fn from_map(map: HashMap<TxId, TransactionRecord>) -> Self {
         TransactionRecordsById(map)
     }
+
+    // modify methods
+
```

Comment:
Please add a unit-test to this method, or reference an open PR that does so?  Same for each new method.

Reply:
#945 

however, this method is not new. just reorganized